config.yml

```diff
@@ -1,38 +1,35 @@
-# Whether to ignore non-FHIR fixtures
+# If true, allow intake non-FHIR fixture (local only)
 nonfhir_fixture: true
 
 # Endpoint against which TestScripts will be executed
 server_url: https://server.fire.ly
 
-# The relative path to the directory containing the TestScript resources (as JSON or XML) to be executed by the engine
-testscript_path: "/spec/examples/testscript_unittest"
+# Relative path to the directory containing the TestScript resources (as JSON or XML) to be executed by the engine
```

Comment:
Currently requires a leading `/` character, which won't be expected. Recommend expecting no leading `/` for this and the testreport_path

Reply:
Good point. Also I don't think we need quotation in config.yml. I will clean it up and recommit.

Reply:
You probably need the quotes if your path contains spaces, otherwise not

Reply:
Right